Test/DatabaseTests/TestProteinReader.cs

```diff
@@ -179,7 +179,16 @@ public static void SeqVarXmlTest()
                 out Dictionary<string, Modification> un);
 
             var target = ok.First(p => !p.IsDecoy);
-            var decoy = ok.First(p => p.IsDecoy);
+            Protein decoy = new Protein(null, null);
```

Comment:
I made this change because ok.First() was sometimes selecting the decoy protein that did not have any sequence variants. This would cause the test to fail. I reworked the test so that the protein selected as the decoy representative always was the one with the sequence variants

Reply:
`var decoy = ok.Where(p => p.IsDecoy && p.SequenceVariants.Count() > 0).First();`